src/relay.js

```diff
@@ -171,13 +179,12 @@ export function sequelizeConnection({name, nodeType, target, orderBy: orderByEnu
     return result;
   };
 
-  let resolveEdge = function (item, args = {}, source) {
-    if (!args.orderBy) {
-      args.orderBy = [defaultOrderBy];
-    }
-
+  let resolveEdge = function (item, index, cursor, args = {}, source) {
+    let startIndex = 0;
+    if (cursor) startIndex = Number(cursor.index);
+    if (startIndex !== 0) startIndex++;
```

Comment:
I'm not following exactly what goes on here, if we receive a cursor we generate a new cursor?